repositories/byod/_posts/2016-01-23-using-private-docker-images.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Using Private Docker Images
+short: Using Private Docker Images
+category: repositories
+order: 7
+---
+
+Many teams use private Docker registries to securely host images that are built from their source code. If your repositories build from private images, you can set up access to your registry so we can build environments with your images.
+
+*Note: This won’t work if you are using Docker Compose files in Runnable. Private registry support for Compose is coming soon.*
+
+---
+
+## 1. Add a File
```

Comment:
I think this would be better worded as "Create Credential File" (or something like that). I was thrown off by the "Add a File" b/c I wasn't sure why I needed to do that to use private images.